### PR TITLE
css-flexbox | ChiaYu Lin

### PR DESCRIPTION
**Pull request by cyulin17**
css-display-flex

css-flex-wrap

css-justify-content

css-align-items

### CODE REVIEW

File: css-flexbox/display-flex.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - display: flex</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="pokemon.css">
+  <style>
+    /* your styles here 👇 */
```
Comment (BrettAlbright) [inline css-flexbox/display-flex.html:12]: you are missing the CSS here.

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
small update!

File: css-flexbox/display-flex.html (hunk 1 above)
Comment (cyulin17) [reply css-flexbox/display-flex.html:12]: Just added. Thank you!

**Review by BrettAlbright — APPROVED** [review]
👍

**Pull request merged**